### ISSUE TITLE
Move changes of waitStatus to upstream

### ISSUE THREAD

**Issue by Katka92** [redhat-developer/che-functional-tests#356]
The timeout for waiting for status of workspace is hardcoded in upstream. This method was overwritten but it would be better to have this feature in upstream. 
Downstream class to remove this method from: RhCheTestWorkspaceServiceClient
Upstream class to change: AbstractTestWorkspaceServiceClient
Method: waitStatus

**Comment by rhopp** [comment]
Once this is done, please create new issue for us to use this in our tests.

**Comment by Katka92** [comment]
PR in upstream https://github.com/eclipse/che/pull/11430

**Comment by dmytro-ndp** [comment]
PR eclipse/che#11430 has been merged with master.

**Comment by Katka92** [comment]
There is only one place where using this method: RhCheTestWorkspaceServiceClient#start. Will solve it as a part of this issue instead of creating new one.

**Comment by Katka92** [comment]
Made changes in upstream. Changes consumed.
